File: MyHeart.Web/ClientApp/models/professionInfo/bonds/SymptomQuestionsSymptomDto.ts
import SymptomDto from "../SymptomDto";

export default class SymptomQuestionsSymptomDto {
    symptomQuestionsId: number;
    symptomId: number;
    bondStrength: number;
    symptom: SymptomDto;

    constructor(symptomQuestionsId: number, bondStrength: number, symptoms: SymptomDto) {
        console.log(symptoms)
        this.symptomQuestionsId = symptomQuestionsId;
        this.symptomId = symptoms.id;
        this.symptom = symptoms;
        this.bondStrength = bondStrength;
    }
    
}

File: MyHeart.DTO/ProfessionInformation/SymptomQuestionBond/SymptomQuestions_SymptomDTO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace MyHeart.DTO.ProfessionalInformation
{
    public class SymptomQuestions_SymptomDTO : BaseEntityDTO
    {
        public int SymptomId { get; set; }
        public int SymptomQuestionsId { get; set; }
        public int BondStrength { get; set; }
        public SymptomDTO Symptom { get; set; }
    }
}
